bears/general/TextLintBear.py

```diff
@@ -0,0 +1,242 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.settings.Setting import typed_list
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='textlint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+)(?:\s|\u2713)*'
+                     r'(?P<severity>error|warning)\s+(?P<message>.+?)'
+                     r'(?:  .*|\n|$)')
+class TextLintBear:
+    """
+    The pluggable linting tool for text and Markdown. It is similar to ESLint,
+    but covers natural language (instead).
```

Comment:
either
`but covers natural language.`
or
`but covers natural language instead.`
Sorry I think I should have been more clear with that :P